### PR TITLE
Prepare release 18.0.0

### PR DESCRIPTION
**Pull request by ericof**
- [X] Updated Description
- [X] Top level Makefile now builds pnpm and yarn images
- [X] Fixed case in Dockefile " as " -> " AS "
- [X] Update GHA to upload description to use the 18.x as the main branch

### CODE REVIEW

File: pnpm/Dockerfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ RUN --mount=type=cache,id=pnpm,target=/app/.pnpm-store,uid=1000 <<EOT
     pnpm install --prod
 EOT
 
-FROM plone/frontend-prod-config:${VOLTO_VERSION} as base
+FROM plone/frontend-prod-config:${VOLTO_VERSION}ASbase
```
Comment (davisagli) [inline pnpm/Dockerfile]: syntax looks wrong, no space (both changed lines in this file)

File: yarn/Dockerfile [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 # Build Volto Project and then remove directories not needed for production
 RUN <<EOT
-    set -e 
+    set -e
     yarn build
     rm -rf cache omelette .yarn/cache
 EOT
 
-FROM plone/frontend-prod-config:${VOLTO_VERSION}-yarn as base
+FROM plone/frontend-prod-config:${VOLTO_VERSION}-yarnASbase
```
Comment (davisagli) [inline yarn/Dockerfile]: same

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -9,7 +9,7 @@ on:
 env:
   BASE_IMAGE_NAME: plone/frontend
   PLATFORMS: linux/amd64,linux/arm64
-  IS_LATEST: false
+  IS_LATEST: true
```
Comment (davisagli) [inline .github/workflows/release.yml:12]: We need to also change this to false on the 17.x branch

Comment (ericof) [reply .github/workflows/release.yml:12]: Just did that

**Review by davisagli — APPROVED** [review]
Looks okay to me now.

**Pull request merged**